sdk/keyvault/azure-security-keyvault-jca/pom.xml

```diff
@@ -151,6 +159,31 @@
                   </excludes>
                 </filter>
               </filters>
+              <relocations>
+                <relocation>
+                  <pattern>org.bouncycastle</pattern>
+                  <shadedPattern>com.azure.security.keyvault.jca.implementation.shaded.org.bouncycastle</shadedPattern>
+                </relocation>
+              </relocations>
+              <transformers>
+                <transformer implementation="org.apache.maven.plugins.shade.resource.ServicesResourceTransformer"/>
+              </transformers>
+            </configuration>
+          </execution>
+          <execution>
+            <id>full-shade</id>
+            <goals>
+              <goal>shade</goal>
+            </goals>
+            <configuration>
+              <shadeTestJar>false</shadeTestJar>
+              <createSourcesJar>${createSourcesJar}</createSourcesJar>
```

Comment:
add `<minimizeJar>true</minimizeJar>`

Reply:
I have added this configuration for the first execution `minimize-shade`, which is only for Bouncy Castle; if I add it for the second execution `minimize-shade`, then the non Bouncy Castle dependencies shaded in the JCA would have breaking changes.